src/components/Navbar.js

```diff
@@ -1,27 +1,36 @@
-import React from 'react'
-import styled from 'styled-components'
-import { Navbar, Nav, Button } from 'react-bootstrap'
+import React from 'react';
```

Comment:
Remove semicolons, as per Standard JS syntax. You can review our readme for project standards.